static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy-one-day-casuals.js

```diff
@@ -5,19 +5,19 @@ import { acquisitionsBannerUsEoyTemplate } from 'common/modules/commercial/templ
 const geolocation = geolocationGetSync();
 const isUS = geolocation === 'US';
 
-const titles = ['It’s because of you...'];
-const messageText = `... and the readers across all 50 states that supported us in 2019 that our journalism thrived in a challenging climate for publishers. Next year America faces an epic choice and the need for an independent press has never been greater. Support from our readers is vital. Please consider supporting us today with a year-end gift. Contribute from as little as $1 and help us reach our goal.`;
+const titles = ['1 day left to give to the Guardian in 2019'];
+const messageText = `… and one year left in Donald Trump’s first term. Over the last three years, much of what the Guardian holds dear has been threatened – democracy, civility, truth. As 2020 approaches, the need for a robust, independent press has never been greater. As we prepare for 2020, we’re asking our US readers to help us raise $1.5 million to cover the issues that matter.`;
```

Comment:
is the '...' meant to be different from the previous one?

Reply:
Good spot! I've had this question in the past. I think when they render on site, they look the same so I stopped paying attention to the difference.

Reply:
👍